src/tools/illink/src/linker/Mono.Linker.csproj

```diff
@@ -76,6 +76,77 @@
     <Compile Include="..\..\external\corert\**\*.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <DependencyAnalysisFrameworkRoot>$(CoreClrProjectRoot)\tools\aot\ILCompiler.DependencyAnalysisFramework</DependencyAnalysisFrameworkRoot>
+    <ToolsCommonRoot>$(CoreClrProjectRoot)\tools\Common</ToolsCommonRoot>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(DependencyAnalysisFrameworkRoot)\ComputedStaticDependencyNode.cs">
```

Comment:
Is there anything blocking a ProjectReference to the ILCompiler project? We use the same project between crossgen2/ilc and it would be preferable here as well.

Reply:
It looks like there was a circular dependency caused by the DependencyAnalysisFramework requiring the trim analyzers via ILLink.Tasks, which references the trimmer. I set _RequiresLiveILLink=false in ILCompiler.DependencyAnalysisFramework.csproj to work around it. Also, the trimmer warned that the assembly wasn't signed, so I set SignAssembly=true.